articles/ams-team-setups.md

```diff
@@ -0,0 +1,43 @@
+
+**ONE TEAM**
+
+![Dummy Placeholder Image](./images/ams/one.png)
+ 
+The ‘One team’ scenario integrates some of the RD scenario benefits – it counts on the whole team being responsible for the application, both for delivery and maintenance. As the team has a shared goal, it ensures a true team spirit. With the team working as one the work gets distributed as it arrives – if an incident occurs in production, the team member with most experience or most available capacity at the given moment takes it over, and if needed with support from the rest of the team brings the issue to conclusion. The knowledge transfer in this case is lean – it is part of the overall project and team work. The team is fully allocated to the project, which means no context switching. A drastic increase of the software quality has been noted, as team members’ goals are the same. Even though this might seam as an ideal situation, there are also some downsides. The capacity impact on the Delivery needs to be considered, even though it is not that high as with other team setups, as well as velocity impact – if there are multiple incidents going on during multiple sprints, the team will reserve capacity to deal with them, subtracting it from the Delivery. This team setup is best applied in situations where the Customer has balanced interest in delivering new features and keeping the production stable.
+The three Live team setups are a product of our learnings and it represents an evolution path – starting from Fixed Developer as a logical and simplest setup and arriving to One Team as a logical conclusion – the project team is one, as the overall goal should be one. We are a part of the project to ensure the success of our Customer in all areas that are in our responsibility.
```

Comment:
Aha! This is more an evolution. Cool. I didn't think of it like that.

Maybe the title of the article could reflect that?
Ideas:
* "DevOps - An Evolutionary Tale"  
* "The Evolution Of Our Operations Team Setup"
* "Things That Change When Transitioning Operation Teams"
* "Something Breaks When You Go DevOps"
* "Operation Teams - The Effects Of Collaboration"